packages/framework-integration-tests/integration/providers/aws/utils.ts

```diff
@@ -11,9 +11,9 @@ import { internet } from 'faker'
 import { sleep } from '../helpers'
```

Comment:
I changed the way we are creating the Apollo clients so that we can now send the token as expected and, besides, we are able to refresh the token